tests/algorithms/lstm_classifier/lstm_classifier_test.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import sys
+import os
+from unittest.mock import MagicMock
+
+sys.path.append(os.path.realpath("."))
+
+import src.algorithms.lstm_classifier.lstm_classifier as lstm_classifier
+
+
+class LstmClassifierTest(unittest.TestCase):
+    def test_get_articles_and_labels(self):
+        review = [
+            {
+                "message": "I just heard about this budgeting app and I gave it a try and done.",
+                "derived-insight": {"category": "Application"},
+            }
+        ]
+        result = lstm_classifier.get_articles_and_labels(review)
+
+        # Check if stop words have been removed from the review message
+        self.assertFalse("and" in result[0][0])
+
+        # Check if review category is the same as label returned
+        self.assertEqual(result[1][0], "Application")
```

Comment:
This test doesn't check the case when `category` has non-word characters (non-alphanumeric). 
you can test with: 
```javascript 
{ 
 "message": "Don't make me sign in everytime.",
 "derived-insight": {"category": "sign-in/sign-up"},
}
```